src/utils/ticketFinder/index.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs').promises;
+const path = require('path');
+
+const getFinderFunctions = async () => {
+  const finderPath = `${__dirname}/finders`;
+  const finderDirs = await fs.readdir(finderPath);
+
+  const finderFiles = await Promise.all(finderDirs.map(async (file) => {
+    const filePath = path.resolve(finderPath, file);
+    const stats = await fs.lstat(filePath);
+    const isDir = stats.isDirectory();
+
+    if (isDir) return file;
+
+    return Promise.resolve();
```

Comment:
Not needed. I was trying to get `reduce` to work, but went with `map` instead. Will remove.